CMakeLists.txt

```diff
@@ -183,13 +194,31 @@ configure_file(
   @ONLY
   )
 
+configure_file(
+  ${KWIVER_CMAKE_DIR}/setup_KWIVER.bat.in
+  ${KWIVER_SETUP_BATCH_FILE}
+  @ONLY
+  )
+
+
 # install set up script
-option( KWIVER_INSTALL_SET_UP_SCRIPT "Whether or not to install setup_KWIVER.sh" ON )
+option( KWIVER_INSTALL_SET_UP_SCRIPT "Creates a setup_KWIVER script (.sh and .bat) that will add properly add kwiver to a shell/cmd prompt" ON )
 mark_as_advanced( KWIVER_INSTALL_SET_UP_SCRIPT )
 
 if( KWIVER_INSTALL_SET_UP_SCRIPT )
   install( PROGRAMS   ${KWIVER_SETUP_SCRIPT_FILE}
     DESTINATION ${CMAKE_INSTALL_PREFIX} )
+  if(WIN32)
+    install( PROGRAMS   ${KWIVER_SETUP_BATCH_FILE}
+      DESTINATION ${CMAKE_INSTALL_PREFIX} )
+  endif()
+endif()
+
+if ( fletch_FOUND )
+  file( APPEND "${KWIVER_SETUP_BATCH_FILE}" "set PATH=%PATH%;${fletch_DIR}/bin\n" )
+  file( APPEND "${KWIVER_SETUP_BATCH_FILE}" "set PATH=%PATH%;${fletch_DIR}/x64/${_vcVersion}/bin\n" )
+
```

Comment:
@dstoup Is there plans to have fletch put the opencv dlls and libs in the appropriate place so projects on windows don't have to have special code to find the appropriate location to add to the path when running?



(Sorry if this is a repeat, not sure what happened to my last conversation)